docs/multicluster/user-guide.md

```diff
@@ -198,52 +198,46 @@ In all clusters, a `ClusterSet` CR must be created to define the ClusterSet, and
 two `ClusterClaim` CRs must be created to claim the ClusterSet and claim the
 cluster is a member of the ClusterSet.
 
-- Create `ClusterClaim` and `ClusterSet` in member cluster `test-cluster-east`
-with the following YAML manifest (you can also refer to
-[multicluster_membercluster_template.yaml](../../multicluster/config/samples/clusterset_init/multicluster_membercluster_template.yaml)):
+- Create `ClusterClaim` and `ClusterSet` in the leader cluster
+`test-cluster-north` with the following YAML manifest (you can also refer to
+[leader-clusterset-template.yml](../../multicluster/config/samples/clusterset_init/leader-clusterset-template.yml)):
 
 ```yaml
 apiVersion: multicluster.crd.antrea.io/v1alpha2
 kind: ClusterClaim
 metadata:
   name: id.k8s.io
-  namespace: kube-system
-value: test-cluster-east
+  namespace: antrea-multicluster
+value: test-cluster-north
 ---
 apiVersion: multicluster.crd.antrea.io/v1alpha2
 kind: ClusterClaim
 metadata:
   name: clusterset.k8s.io
-  namespace: kube-system
+  namespace: antrea-multicluster
 value: test-clusterset
 ---
 apiVersion: multicluster.crd.antrea.io/v1alpha1
 kind: ClusterSet
 metadata:
   name: test-clusterset
-  namespace: kube-system
+  namespace: antrea-multicluster
 spec:
   leaders:
     - clusterID: test-cluster-north
-      secret: "member-east-access-token"
-      server: "https://172.18.0.1:6443"
-  members:
-    - clusterID: test-cluster-east
-  namespace: antrea-multicluster
```

Comment:
@luolanzone : I assume `namespace` is not needed in leader ClusterSet spec. Could you confirm?

Reply:
I saw you mentioned we still validate it in leader. Could we remove the check in the leader cluster?

Reply:
Do you mean to remove the Namespace checking in the leader controller when there is a ClusterSet created in the leader Namespace? I submitted a PR for this https://github.com/antrea-io/antrea/pull/4074, could you check?